base/src/colors.ts

```diff
@@ -67,6 +67,75 @@ export interface Theme {
     country_label_halo: string;
 }
 
+export const BASIC: Theme = {
+    hasCasings: true,
+    background: "#dddddd",
+    earth: "#ededed",
+    park: "#c2f7d1",
```

Comment:
Consider using `#bfc99c` here. Otherwise parks stil use the bright green:

<img width="598" alt="Screenshot 2023-06-16 at 22 52 10" src="https://github.com/protomaps/basemaps/assets/795927/699906d7-fbe9-4ac5-a88a-b2aa27863841">

Reply:
done!